phpBB/phpbb/attachment/upload.php

```diff
@@ -296,26 +307,6 @@ protected function check_attach_quota()
 	 */
 	protected function check_disk_space()
 	{
-		if ($free_space = @disk_free_space($this->phpbb_root_path . $this->config['upload_path']))
-		{
-			if ($free_space <= $this->file->get('filesize'))
-			{
-				if ($this->auth->acl_get('a_'))
-				{
-					$this->file_data['error'][] = $this->language->lang('ATTACH_DISK_FULL');
-				}
-				else
-				{
-					$this->file_data['error'][] = $this->language->lang('ATTACH_QUOTA_REACHED');
-				}
-				$this->file_data['post_attach'] = false;
-
-				$this->file->remove();
-
-				return false;
-			}
-		}
-
```

Comment:
So.. you just assume that there always is disk space available?

Reply:
For now... yes, this has to be fixed after the pr track storage files